### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by benib**
- updates node and all dependencies
- fixes the issue with the prognosisPattern not being visible on bars and columns

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,41 +5,45 @@
...
-    "node-fetch": "^1.7.3",
+    "boom": "^7.3.0",
+    "canvas": "^2.5.0",
+    "clone": "^2.1.2",
+    "d3-array": "^2.2.0",
+    "d3-format": "^1.3.2",
+    "d3-time-format": "^2.1.3",
+    "decimal.js": "^10.2.0",
+    "deepmerge": "^3.2.1",
+    "hapi": "^18.1.0",
```
Comment (manuelroth) [inline package.json]: Are these dependencies still needed? (hapi, boek, inert)- Because we now use the vendor-prefixed ones?

Comment (benib) [reply package.json]: of course not, thanks for catching this! removed with next commit.

File: package.json [hunk 2]
```diff
@@ -5,41 +5,42 @@
...
-    "d3-format": "^1.2.1",
-    "d3-time-format": "^2.1.1",
-    "decimal.js": "^10.0.1",
-    "deepmerge": "^2.0.1",
-    "hapi": "^17.8.3",
-    "hoek": "^5.0.2",
-    "inert": "^5.0.1",
-    "moment-timezone": "^0.5.16",
-    "node-fetch": "^1.7.3",
+    "boom": "^7.3.0",
```
Comment (manuelroth) [inline package.json]: `boom` should be removed as well

Comment (benib) [reply package.json]: true, done.

**Pull request merged**